.travis/script.sh

```diff
@@ -29,6 +29,16 @@ end_section "info.conda.config"
 
 $SPACER
 
+start_section "symbiflow.run_python_tests" "Run Python unit tests"
+# Reset working directory
+pushd $TRAVIS_BUILD_DIR
+pip install -r ./requirements.txt
+python -m unittest utils.lib.rr_graph.tests.test_graph
```

Comment:
Should we use pytest here?  I don't think we explicit specific python paths on the prjxray python tests?

Reply:
This change is not addressing that. It is just moving them earlier in travis script and using an environment variable instead of a hard-coded path so it will pass on forks.

We need to address #463 before we can start calling all the tests. We should also have them run as build target.